src/components/Header.js

```diff
@@ -91,18 +85,21 @@ class Header extends Component {
   render() {
     const { session, navigation } = this.props
     return (
-      <Navbar inverse={true} id="nav_header">
+      <Navbar inverse={true}>
         <Navbar.Header>
           <Navbar.Brand>
             <LinkContainer to={ROUTE_ROOT}>
               <img src={logo} className="App-logo" alt="logo" />
             </LinkContainer>
           </Navbar.Brand>
+          <Navbar.Toggle />
         </Navbar.Header>
         <Navbar.Collapse>
           {this.renderNavigation(navigation, session.isAnonymous)}
```

Comment:
when these are clicked, can we close the menu?
 
Right now, I am seeing it stay open
![2019-02-27 10 55 09](https://user-images.githubusercontent.com/5168796/53515199-4e921e80-3a7e-11e9-9fae-df68b36f0cc7.gif)